apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -912,3 +916,61 @@ i.infoAlertIcon {
 .detailsHidden {
   display: none;
 }
+
+.studentDropdownOptionContainer {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+}
+
+.studentDropdownOption {
+  display: flex;
+  flex-direction: row;
+  gap: 5px;
+  margin-top: 3px;
+}
+
+.greenStatusBlob {
+  background-color: $light_affirmative_100;
+  border-radius: 20px;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-top: 4px;
+  display:inline;
+  p {
+    margin-bottom: 0;
+  }
+}
+
+.yellowStatusBlob {
+  background-color: $lighter_yellow;
+  border-radius: 20px;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-top: 4px;
+  display:inline;
+  p {
+    margin: 0,0,0,0;
+  }
+}
+
+.grayStatusBlob {
+  background-color: $light_gray_100;
+  border-radius: 20px;
+  padding-left: 10px;
+  padding-right: 10px;
+  padding-top: 4px;
+  display:inline;
+  p {
+    margin: 0,0,0,0;
+  }
+}
```

Comment:
Check the bubble styling in the figma. The text looks a little off. You may need to modify the padding-top to bump it down to the center.

Reply:
The height may also need to be set to a constant value.

Reply:
Updated the styling
<img width="274" alt="Screenshot 2024-03-06 at 1 52 52 PM" src="https://github.com/code-dot-org/code-dot-org/assets/7144482/598c44e9-9eb7-4400-b6b3-fd901b36dbc7">

Reply:
lol I forgot to push, but it's there now
<img width="263" alt="Screenshot 2024-03-06 at 2 58 42 PM" src="https://github.com/code-dot-org/code-dot-org/assets/7144482/cd2e2be1-aad6-4f79-988e-7253c1046387">